src/api/config/routes.rb

```diff
@@ -732,8 +732,9 @@ def self.public_or_about_path?(request)
 
   # StagingWorkflow API
   scope module: 'staging' do
-    resources :staging_projects, only: [:show], param: :name do
+    resources :staging_projects, only: [:show, :index], param: :name do
       get 'staged_requests' => 'staged_requests#index', constraints: cons
+      get 'staged_projects' => 'staged_projects#index', constraints: cons
```

Comment:
Why adding the route twice? (I know they're going to two different controllers but you're only adding one controller)